src/test/java/step3/domain/CarsTest.java

```diff
@@ -0,0 +1,38 @@
+package step3.domain;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class CarsTest {
+
+  public static final int NUMBER_OF_CAR = 5;
+  private Cars cars;
+
+  @BeforeEach
+  void before() {
+    cars = new Cars(NUMBER_OF_CAR);
+  }
+
+  @DisplayName("모두 이동 가능한 조건이라, 이동한 경우")
+  @Test
+  public void canMoveIsTrue() {
+    cars.race(new TestAllTrueRacingRule());
```

Comment:
TestAllTrueRacingRule 클래스 추가 좋네요 👍 
그런데 canMove를 테스트하기 위해 true, false를 고정한 클래스를 만들다보면 관리해야될 클래스가 많아지겠네요
중복을 제거하기 위한 아래와 같은 방법도 있으니 참고하세요

```suggestion
    cars.race(createTestRacingRule(true));
```

```java
  // CarsTest 클래스 내에 메서드 추가
  private RacingRule createTestRacingRule(boolean canMove) {
    return new RacingRule() {
      @Override
      public boolean canMove(int targetNumber) {
        return canMove;
      }

      @Override
      public int getTargetNumber() {
        return 0;
      }
    };
  }
```